Classes/Service/OAuthService.php

```diff
@@ -141,7 +141,7 @@ protected function getProvider()
                 'urlAuthorize' => $this->settings['oidcEndpointAuthorize'],
                 'urlAccessToken' => $this->settings['oidcEndpointToken'],
                 'urlResourceOwnerDetails' => $this->settings['oidcEndpointUserInfo'],
-                'scopes' => ['openid'],
+                'scopes' => $this->settings['oidcScopes'],
```

Comment:
oups, and actually this one as well, you need to combine everything that fit together into a single commit and put a meaningful commit message like `[TASK] Add support for OIDC scope` since there is no point into writing that you update file X or Y, we know that by reading the commit log, but what the commit actually "does" is what is important with a commit message.